packages/react-hooks/src/useBrokerSalesInfo.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017-2024 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { PalletBrokerSaleInfoRecord } from '@polkadot/types/lookup';
+import type { PalletBrokerSaleInfoRecord as SimplifiedPalletBrokerSaleInfoRecord } from './types.js';
+
+import { useEffect, useState } from 'react';
+
+import { createNamedHook, useApi, useCall } from '@polkadot/react-hooks';
+
+function parsePalletBrokerSaleInfoRecord (record: PalletBrokerSaleInfoRecord): SimplifiedPalletBrokerSaleInfoRecord {
+  return {
+    coresOffered: record.coresOffered.toNumber(),
+    coresSold: record.coresSold.toNumber(),
+    endPrice: record.endPrice,
+    firstCore: record.firstCore.toNumber(),
+    idealCoresSold: record.idealCoresSold.toNumber(),
+    leadinLength: record.leadinLength.toNumber(),
+    regionBegin: record.regionBegin.toNumber(),
+    regionEnd: record.regionEnd.toNumber(),
+    saleStart: record.saleStart.toNumber(),
+    selloutPrice: record.selloutPrice.isSome ? record.selloutPrice.unwrap() : 0
```

Comment:
Small nit, but I think you can set the `0` to `new BN(0)` if you want the values to be inline. That being said its a small nit.

Reply:
good nit 🤩